install_config/persistent_storage/persistent_storage_glusterfs.adoc

```diff
@@ -13,19 +13,52 @@ toc::[]
 
 == Overview
 
-{product-title} clusters can be provisioned with
-xref:../../architecture/additional_concepts/storage.adoc#architecture-additional-concepts-storage[persistent storage]
-using GlusterFS.
-
-Persistent volumes (PVs) and persistent volume claims (PVCs) can share volumes
-across a single project. While the GlusterFS-specific information contained in a
-PV definition could also be defined directly in a pod definition, doing so does
-not create the volume as a distinct cluster resource, making the volume more
-susceptible to conflicts.
-
-This topic presumes some familiarity with {product-title} and
-https://access.redhat.com/documentation/en-US/Red_Hat_Storage/3/html/Administration_Guide/index.html[GlusterFS].
-See the xref:../../architecture/additional_concepts/storage.adoc#architecture-additional-concepts-storage[Persistent
+You can configure your {product-title} cluster to use Red Hat Gluster Storage as
+persistent storage for containerized applications. There are two deployment
+solutions available when using Red Hat Gluster Storage, using either a
+containerized or dedicated storage cluster. This topic focuses mainly on the the
+persistent volume plug-in solution using a dedicated Red Hat Gluster Storage
+cluster.
+
+[[gfs-containerized-storage-cluster]]
+=== Containerized Red Hat Gluster Storage
```

Comment:
I debated whether it seemed weird to have "Containerized Red Hat Gluster Storage" and "Dedicated Storage Cluster" be subsections under "Overview", which we typically avoid. I wanted the containerized option to be clearly visible from the TOC, but not be so prominent (i.e., same level as Overview) that it looks like a step along the way of the dedicated solution.